2015/misc/inputs/C.in

```diff
@@ -0,0 +1,4 @@
+3
+1 3 5
+2 4 7
+1 1 0
```

Comment:
Please remove all these duplicated input files. All input files should be placed at `misc/inputs` directory